src/css/controls/imports/nextup.less

```diff
@@ -163,3 +163,7 @@
         visibility: visible;
     }
 }
+
+.jw-nextup-firefox-pip-fix {
+    background: fade(@overlay-background-color, 100%);
```

Comment:
No need to use fade here. It's a static number. LESS functions are useful for mixins and variables but for a static value it's overkill. Feel free to leave it in as this is more of a nit, but there's no advantage to using fade here.